meta-facebook/yv35-hda1/src/platform/plat_isr.c

```diff
@@ -116,15 +131,30 @@ void ISR_MPRO_BOOT_OK()
 	}
 }
 
+bool last_hb = false;
```

Comment:
What is the purpose of this variable? Didn't see it being used anywhere.

Reply:
Got it, thanks. it will be removed in next commit.